src/app/static/src/components/parameters/EditPhases.vue

```diff
@@ -3,7 +3,7 @@
     <modal class="phase-modal" :open="open">
       <h3>Edit {{paramGroup && paramGroup.label}}</h3>
       <div class="mb-3">
-        Click on a Phase to drag it to a new start date.
+        Click on a Phase to drag it to a new start date. Click on timeline to add a new Phase.
```

Comment:
This could say "the timeline". Regardless, I had to think twice about what the timeline was: at first I assumed thin horizontal line. But then to my relief I realised I could click anywhere. Sorry - not constructive criticism, more an observation!

Reply:
I think it should be more obvious once the dates are added to the timeline, perhaps, but I may tweak the text anyway.

Reply:
..is 'above the timeline' better do you think?